src/Service/Resolvers/Sql Query Structures/SqlUpsertQueryStructure.cs

```diff
@@ -195,6 +204,7 @@ public ColumnDefinition GetColumnDefinition(string columnName)
         private void PopulateColumnsAndParams(string columnName)
         {
             InsertColumns.Add(columnName);
+            FieldsReferencedInDbPolicyForCreateAction.Remove(columnName);
```

Comment:
>FieldsReferencedInDbPolicyForCreateAction [](http://example.com/codeflow?start=12&length=41)

perhaps a comment describing why this is necessary. per your previous pr 1391's description: this is necessary because any leftover fields will result in exception because they weren't included in the request.